asteroid/dsp/spatial.py

```diff
@@ -0,0 +1,65 @@
+import torch
+import torch.nn.functional as F
+
+
+def xcorr(ref, input, normalized=True, eps=1e-8):
```

Comment:
Named after the Matlab function ?

Reply:
maybe, i named that way only because it sounds cooler i did not think about MATLAB

Reply:
It sounds cool! :smiley: